bin/install-checkstyle.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# use `make upgrade` to update this URL to the latest version
+URL='https://downloads.sourceforge.net/project/checkstyle/checkstyle/8.2/checkstyle-8.2-all.jar?r=&ts=1505330021&use_mirror=razaoinfo'
```

Comment:
@maxjacobson WDYT?

Reply:
what's the deal with the query string? Can we leave it off?

Reply:
@maxjacobson Seems to be how it selects the mirror. Testing without it.

Reply:
@maxjacobson It works. Updated to take it off.